rebar.config.script

```diff
@@ -15,6 +15,18 @@
 ConfigureEnv = filename:join(filename:dirname(SCRIPT), "config.erl"),
 os:putenv("COUCHDB_CONFIG", ConfigureEnv).
 
+OTP_VSN = erlang:system_info(compat_rel),
+
+Features = lists:filter(fun({_K, Flag}) -> Flag end, [
+    {'HAS_CALLBACKS', OTP_VSN > 14}
+]),
+
+Properties = Features ++ [
+    {'OTP_VSN', OTP_VSN},
+    {'OTP_RELEASE', erlang:system_info(otp_release)}
+],
```

Comment:
The reason I didn't hardcoded defines as `[{d, Key, Value}]` is to be able to define some macro from the content of `config.erl`. But I didn't do a second step to actually use `config.erl`. I'll update the PR with @rnewson's solution.